content/50.discussion.md

```diff
@@ -11,8 +11,42 @@ In this review, we seek to explain the scientific rationale underlying these tec
+The racism within medicine is critical to understanding how practitioners and researchers’ bias negatively affects health outcomes.
+There is a misconception that African-American individuals have thicker skin and exorbitant pain tolerance in comparison to other demographics.
+As a result, African-Americans can be misdiagnosed and be given incorrect prescriptions.
+The glomerular filtration rate (GFR) is a tool used to measure a specific protein in the blood, which is an indicator of kidney function.
+This sophisicalted equation inputs data, such as, weight, age, sex, serum levels of creatinine, and race, to obtain a result.
+Black individuals have a higher racial adjustment factor than that of other races.
+Thus, with similar factors as a white person, for instance, a Black person would have greater kidney function, leading to more kidney complications to reach the threshold of kidney disease.
+In addition to misdiagnosis, participation in clinical trials by African-Americans is disproportionately lower than that of caucasians, even in diseases which [African-Americans] are more susceptible.When there is equal opportunity to participate in clinical trials, there is an insignificant difference between racial groups.
+
+Why are certain racial groups disproportionately being infected with Covid-19.
+Studies suggest that discussions between doctors and patients impact whether or not a physician offers joining a clinical trial.
+The results of the studies convey that physicians’ visits with African-Americans were shorter and provided less information about clinical trials than that of whites.
+Some of the reasons for this disparity are lack of awareness and education, mistrust in healthcare professionals, and lack of health insurance because otherwise there should be no disparity.
+These studies show that both physicians and patients have a bias or misinformation that can affect health outcomes.
+
+There is little to no data on the ongoing clinical trials for COVID-19, however, one can presume that African-Americans and other racial minorities are not obtaining equitable access to clinical trials for a disease that research has shown disproportionately affects them.
+While much of the research on health disparities and bias are inconclusive, humans, specifically scientists in medicine, need to make an effort to think more global and be more inclusive with their clinical trials to aid the health outcomes of all demographics.
+
+<!--From #460: Risk of comorbid health conditions associated with more severe outcomes may be influenced by long-term damage caused by chronic stress related to traumatic social experiences [@doi:10.1007/s12170-013-0338-5], perhaps mediated by cardiovascular risk factors [@doi:10.1101/2020.05.10.20097253], although the effects of chronic stress have not yet been researched in the specific case of COVID-19 disparities.-->
```

Comment:
@tlukan, this was suggested previously by another contributor. Do you think it would be an interesting topic to expand on alongside what you are developing? Previously I wanted to study allostatic load (long-term effects of chronic stress), so I have definitely got some materials I put together that I could use to make an outline in case it would be helpful. I do think your clinical trial participation point probably has the most direct effect on what we're analyzing, though.

Reply:
I do agree that we should say that it's important to make sure that people in racial minorities and people in lower-income jobs have ample opportunity to be represented in trials, especially since they are at the highest risk.  But I think it would be better to present this specifically in the context of the implications for COVID-19, because the paper as a whole isn't about the sociology of medicine.

Reply:
Thank you @dziakj1 I will make the point more specific to covid-19. But much information about racial bias in covid-19 clinical trails isn't available to my knowledge, so should I present prior research on other diseases and allude to the fact that covid-19 is applicable to these issues?  Since the paper is about covid I should be general in discussing prior research?